components/datetime/src/provider/neo.rs

```diff
@@ -85,7 +119,10 @@ pub enum MonthSymbolsV1<'data> {
 #[icu_provider::data_struct(
     marker(WeekdaySymbolsV1Marker, "datetime/symbols/weekdays@1"),
     marker(DayPeriodSymbolsV1Marker, "datetime/symbols/dayperiods@1"),
-    marker(DaySymbolsV1Marker, "datetime/symbols/days@1")
+    marker(ChineseDaySymbolsV1Marker, "datetime/symbols/chinese/days@1"),
+    marker(DangiDaySymbolsV1Marker, "datetime/symbols/dangi/days@1"),
+    // for calendars that don't use day symbols
+    marker(PlaceholderDaySymbolsV1Marker, "datetime/symbols/placeholder/days@1"),
```

Comment:
Nit: Please open an issue to bikeshed the name of this

Reply:
https://github.com/unicode-org/icu4x/issues/4186